### ISSUE TITLE
[BUG] the google play store updated and get version not work fine

### ISSUE THREAD

**Issue by exbein** [JoMingyu/google-play-scraper#144]
**google_play_scraper.VERSION**
1.2.1

**Describe the bug**
this is resp about get pokemon go 
```
{'title': 'Pokémon GO', 'description': 'New! Now you can battle other Pokémon GO Trainers online! Try the GO Battle League today!\r\n\r\nJoin Trainers across the globe who are discovering Pokémon as they explore the world around them. Pokémon GO is the global gaming sensation that has been downloaded over 1 billion times and named “Best Mobile Game” by the Game Developers Choice Awards and “Best App of the Year” by TechCrunch.\r\n_______________\r\n\r\nUncover the world of Pokémon: Explore and discover Pokémon wherever you are!\r\n \r\nCatch more Pokémon to complete your Pokédex!\r\n \r\nJourney alongside your Buddy Pokémon to help make your Pokémon stronger and earn rewards!\r\n\r\nCompete in epic Gym battles and...\r\n\r\nTeam up with other Trainers to catch powerful Pokémon during Raid Battles!\r\n \r\nIt’s time to get moving—your real-life adventures await! Let’s GO!\r\n_______________\r\n\r\nNotes: \r\n- This app is free-to-play and offers in-game purchases. It is optimized for smartphones, not tablets.\r\n- Compatible with Android devices that have 2GB RAM or more and have Android Version 6.0–10.0+ installed.\r\n- Compatibility is not guaranteed for devices without GPS capabilities or devices that are connected only to Wi-Fi networks.\r\n- Application may not run on certain devices even if they have compatible OS versions installed.\r\n- It is recommended to play while connected to a network in order to obtain accurate location information.\r\n- Compatibility information may be changed at any time.\r\n- Please visit PokemonGO.com for additional compatibility information. \r\n- Information current as of October 20, 2020.', 'descriptionHTML': 'New! Now you can battle other Pokémon GO Trainers online! Try the GO Battle League today!<br><br>Join Trainers across the globe who are discovering Pokémon as they explore the world around them. Pokémon GO is the global gaming sensation that has been downloaded over 1 billion times and named “Best Mobile Game” by the Game Developers Choice Awards and “Best App of the Year” by TechCrunch.<br>_______________<br><br>Uncover the world of Pokémon: Explore and discover Pokémon wherever you are!<br> <br>Catch more Pokémon to complete your Pokédex!<br> <br>Journey alongside your Buddy Pokémon to help make your Pokémon stronger and earn rewards!<br><br>Compete in epic Gym battles and...<br><br>Team up with other Trainers to catch powerful Pokémon during Raid Battles!<br> <br>It’s time to get moving—your real-life adventures await! Let’s GO!<br>_______________<br><br>Notes: <br>- This app is free-to-play and offers in-game purchases. It is optimized for smartphones, not tablets.<br>- Compatible with Android devices that have 2GB RAM or more and have Android Version 6.0–10.0+ installed.<br>- Compatibility is not guaranteed for devices without GPS capabilities or devices that are connected only to Wi-Fi networks.<br>- Application may not run on certain devices even if they have compatible OS versions installed.<br>- It is recommended to play while connected to a network in order to obtain accurate location information.<br>- Compatibility information may be changed at any time.<br>- Please visit PokemonGO.com for additional compatibility information. <br>- Information current as of October 20, 2020.', 'summary': 'Discover Pokémon worldwide', 'installs': '100,000,000+', 'minInstalls': 100000000, 'realInstalls': 447695301, 'score': 4.28665, 'ratings': 15311870, 'reviews': 976920, 'histogram': [1464232, 419982, 886752, 2032315, 10508574], 'price': 0, 'free': True, 'currency': 'USD', 'sale': False, 'saleTime': None, 'originalPrice': None, 'saleText': None, 'offersIAP': True, 'inAppProductPrice': '$0.99 - $99.99 per item', 'developer': 'Niantic, Inc.', 'developerId': 'Niantic,+Inc.', 'developerEmail': 'pokemon-go-support@nianticlabs.com', 'developerWebsite': 'https://niantic.helpshift.com/a/pokemon-go/?p=web', 'developerAddress': 'One Ferry Building, Suite 200\nSan Francisco, CA 94111', 'privacyPolicy': 'https://nianticlabs.com/privacy/pokemongo/en', 'genre': 'Adventure', 'genreId': 'GAME_ADVENTURE', 'icon': 'https://play-lh.googleusercontent.com/SVQIX_fYcu5mc4Pq-D7dgxXZdRMpNTAbRKeBJygAsIXKITHEcKckyhzLsIXMQLSRZw', 'headerImage': 'https://play-lh.googleusercontent.com/KgDQ-Kjb2B7_jDP-8KmQDNhAmP2lqAV_w3zArOCBL7YZnQ02Qqp4VTlgdocO-4MFk4s', 'screenshots': ['https://play-lh.googleusercontent.com/O-OR6Mh0AoNyiaYYaa3OJ_VHGfLqWW2qNzUUZxRRodD3fqs2Pm04FatavdNbz-jsMZM', 'https://play-lh.googleusercontent.com/HiLNXxmIpOl1jLBssHWcGqsQ58oUC5RvmS5tiX7L86mPHCG6wVEN-aX5OxTKAbDzh10', 'https://play-lh.googleusercontent.com/Il9B11lqrHX_Kd3QzLCA6hA6O7EsT56ItiLWMf1JkwcdmRyR3CE2KW_vR9w1izanO-JM', 'https://play-lh.googleusercontent.com/_84Pq9dJ0_iTKwx9-CkYFYoakSMKK_yfdrp1WSX-inE2XvCTr8ri2tKoISLHN_9PEvP5', 'https://play-lh.googleusercontent.com/JjgLXR7CMiQfSbtICOa86_35f7Pf8IzJn78zWQslwJn56Qm2O7BOV7xzXXE8mz3Vhg', 'https://play-lh.googleusercontent.com/iAYd9LdbFI7aajPU760XoNZ8b3woDQ58B0harYiUed2y7WQLU19fcj9I8yS-_K9BDQ', 'https://play-lh.googleusercontent.com/lpw5tz7Onf_6Sx4Q3kGX1zKXcec4EWpRyr9I4w5d3TrQMoorPWVke6veB5qmqhfQZn4'], 'video': 'https://www.youtube.com/embed/DFXbVBFPOOs?ps=play&vq=large&rel=0&autohide=1&showinfo=0', 'videoImage': 'https://play-lh.googleusercontent.com/KgDQ-Kjb2B7_jDP-8KmQDNhAmP2lqAV_w3zArOCBL7YZnQ02Qqp4VTlgdocO-4MFk4s', 'contentRating': 'Everyone', 'contentRatingDescription': 'Mild Fantasy Violence', 'adSupported': False, 'containsAds': False, 'released': 'Jul 6, 2016', 'updated': 1660077582, 'version': 'Varies with device', 'recentChanges': 'Here’s what’s new in Pokémon GO!\r\n\r\n- A new type of Incense is coming to  Pokémon GO: Daily Adventure Incense! Who knows what Pokémon you’ll encounter...\r\n', 'recentChangesHTML': 'Here’s what’s new in Pokémon GO!<br><br>- A new type of Incense is coming to  Pokémon GO: Daily Adventure Incense! Who knows what Pokémon you’ll encounter...<br>', 'comments': ["it was the best thing in the world in 2016. but for a game that needs to update every time I open it, it sure hasn't improved a lot. theres been some additions and changes. but overall the game is still basic. all thr small things like animations between screens that take forever and unresponsive notifications or taps in general, etc. tons. those small things have driven me crazy for the past 6 years. and not a single uodate to make things more efficient and smooth. seems like a cash grab now.", 'Still really fun. Started playing 2 months ago for the first time in years. There\'s handful of new features that polish it nicely. Ideas: it\'d be cool if players could experience different "research" questlines instead of the same one, maybe by asking the player to make plot-affecting choices. I\'d also suggest less churches be pokestops and more situationally appropriate buildings like hospitals or grocery stores. A funeral home is a pokestop near me, it feels weird to send gifts from there', 'Love this game and play it probably more than I should. I played when it first came out but stopped playing shortly after. Around 2018 I downloaded it again and have been hooked ever since. You will benefit playing in a larger city with more people over a smaller town. I have had some issues and bugs too. The most annoying one currently is for some reason the screen dims while using the app regardless of phone settings. Raids will kick people out randomly and you have to contact support.', 'I would say the experience is as good as you have room for the game. The game is fun and gives me a reason to get out of the house and explore towns around me. But the real stickler of the game is how much space it needs especially with constant updates. It becomes a bit unbarable when you have to constantly delete apps to make room. Best to get a micro sd card for your phone to make it worth playing more.', "Overall very enjoyable and have spent many hours on it. It has room for improvement but with I feel are pretty frequent updates and changes my interest is still held. I started my account back in 2016 and in comparison to then it has become drastically more enjoyable to play, is very addictive, and love that it encourages walking. Tho when taking a brake it's very hard to get back into the groove and accidentally breaking the streak (even if it's literally is by 5 minutes) kills motivation.", 'Updates either too demanding of space or time needed. Lags with slow internet hard. Events are often lacking true substance. Plus level 40 is for hardcore fans. Not saying it should be too easy but people do have lives outside the game. Edit: the bugs and glitches are just so much fun. Its great to oem the app to a blank screen which cant really be fixed until you cam actually open the app.', "Overall it is a balanced game fun and challenging. Trying to get you to get out side and have fun while exercising. I feel as tho most people play while driving which isn't as fun as walking with friends. Walking with my pet is an adventure so is battling other people, I hope they come out with greater updates but an update every month isn't necessary.", "It's a good game, but everything takes way too long. Seems petty, but it ruins the fun quickly. Every animation takes so ridiculously long. Catching pokemon, raiding, opening gifts, and every little thing in-between. I want to play the game, not stand idle in the middle of town waiting on unnecessarily slow animations to come to a finish. 5-star rating, pending on some easily implemented skip buttons. If it's our time you want, Niantic, listen to us and you'll get more of it in return. Duh.", "Very fun game I've consistently gone back to. The level of engagement offered is greatly appreciated and well rivals what's *currently* on the market. Disappointing lack of development on the games main selling point, being AR functionality, and a disappointing lack of traditional thematic elements to make up for it. Overall a good product that sets a standard for the product category. If only that standard was set a little higher.", 'Absolutely loved it, but the "upgrades" lately are downgrades. Remote raid passes are no longer one freebie a week, AND they raised the prices of purchasing them! I know many trainers are unhappy with this, as we don\'t always have time, energy, or enough people to walk for hours though town and meet up. I know it\'s a "go outside and get exercise" game, which is fun, but not always achievable, especially when you already have a physically demanding job.', "It keeps crashing to the point that it interferes with playability. This started just under a month ago, and it has only just begun to be slightly playable again, but it doesn't go more than a few minutes before it crashes again, and it has high correlation with certain activities, like initiating a battle in a gym or opening gifts from friends. Please fix this.", "God I wish you guys would allow custom start and end times for events. where I live in the summer there's always an excessive heat warning and it seems most of your events like to take place during the height of that time. Possibly adding at least the ability to pause once for emergency or you know weather based warnings. Best times to play where I live would be very early morning or late at night around sunset or after the sun has gone down. Otherwise it's playing doors.", "I've been playing this since 2016, off and on. Took a few months long break and decided to reinstall it when I got a new phone. Everything worked fine in the past but now none of my eggs will make any progress while incubating, neither will the buddy adventure distance. What gives? I gain distance using adventure sync but nothing else shows up? I have tried all troubleshooting on my Samsung Galaxy S22+ and nothing has worked. Someone please reach out to me so I can have this fixed!!!", "I've been playing since the begining of the game. I loved the game when first came out. Some changes and updates to the game made it more interesting and some others were not so great. Now they took out the two stops closet to my home one block second 2 blocks. Those i was able to get to witout much trouble. The third one closest you have to jump 2 sets of tracks and walk through tall grass and weeds. Its hard to get to and dangerous for the players. There is barely a signal in the counrty", "It can be slow, and even shut off, but it is immensely better than before. I do love this app. However, for an app that is involving AR more and more, especially for the buddy, you would think it'd actually work. The AR camera is garbage beyond belief. Even in Pikmin Bloom it's trash, yet it tends to be an important component. I love the game, there's a lot to like. I can't give it 4 stars though when there is a problem that has been neglected.", "I love this game. The only problem I have is that it won't allow me to scan with AR mapping for an A12 Samsung. It allows for versions below and above but skips a select few in the middle for some reason. Also, battles are very laggy. I love the fact that you can catch pokemon, connect with others, earn rewards and customize your character. If you fix the technical aspect ots def 5/5", "Everything is fine but the keyboard won't stay up when I want to change my Pokémon's name. Most of the time it won't even come up, it's just the result of the keyboard and not the whole thing. It's bothersome and I don't know if it's my phone or the app itself. Also whenever I try to turn on adventure sync it kicks me out of the app and sends me straight to the home screen. Update; I reinstalled the app, only to find after pressing on a Pokemon to catch, it instead goes to a white screen.", "After returning to the game I just see a lot of bugs and random lag. Plus it clearly caters to people in a more populated setting. I tried to get back into it but it's just not worth. And seriously why is there no variety in the Team Rocket shadow Pokémon you can catch? I'm so sick of seeing the stupid balloons every ten seconds because I don't even want to bother with them anymore. So boring.", "Fun app overall. Wishlist to improve game experience: 1. Would like to be able to open more gifts daily. 2. Remote raids are glitchy at times. It kicks me out at the go screen. 3. Would be great if we could spin pokestops that are visible on the board. Can't always go out but want to keep streaks. 4. Most importantly, I wish we could trade w/ any friend. At least at a certain friendship or trainer level, minimally for lucky friends. I have no friends locally, but friends all over the world.", "*Edit#2* Changed it to one from three stars. Issues I had before were fixed but now after I was logged out I can't log back into my account. I've retrieved my username and changed my password multiple times and it was all for nothing. I got so frustrated that I decided that I just wanted to delete my account so I can start from scratch. Can't even do that because app support doesn't seem to be a thing even after jumping through so many hoops to find it. I might be done with it forever now", "I've always loved Pokemon Go. Although I sometimes don't play it for a couple months, when I get back in the habit, it's fun. The game has so many nice features and it's easy to play and everything. My only issue as of right now is the storage it takes. I don't have much storage on my phone so having to delete apps every single time there's an update gets annoying because you physically cannot play unless updating. Other than that, amazing game.", 'The recent changes to the QOL updates they made prior to the pandemic have made the game yet again hard to get into if you dont have access to populated areas or dont have transport. The last 2 paid events had "increased" shiny chances which were rather lackluster, as well as both were immensely grind heavy and tedious. If your a massive Pokemon fan or have been playing for awhile or live in the middle of a city, you prolly will enjoy, if not, pass it up.', 'Program uses alot of space. Constantly restarting while playing. Walk buddies not updating correctly. Seriously considering quitting this game. More issues and restrictions are coming out then their fixing. Why are gold metals before the platinum medals? Break them into a different category. Platinum is the highest medal (platinum gold silver bronze)', "Been playing Pokémon for over two decades. Very nostalgic of the first two generations with a modernized mechanic when it comes to the battle/catching system. The level up process may irk some of the handheld game fans, as you level up through tasks/candies rather than through battles..but the game is fairly in depth. I've traveled across 5 states playing this game, and i'm still having fun four years later!", "This has been my favorite game since 2016 and has only gotten better since. I only have one issue with the game play. One, during a battle I will usually charge up my charged attack and then press it repeatedly and I still don't get to use it. This happens only a few times every once and a while but when it does I lose almost every time without fail.", "Solid game, sometimes has connection issues, errors, and lag, but that's fairly rare these days. Big curve when it comes to getting stronger Pokémon; takes a long time, especially if you want to do GO Battle League. Trading system is annoying but it's not necessary for completion outside region locked 'mons. Good game to play while waiting for something or just to burn time, and a very different experience from mainline Pokémon games.", "It's fun, however there are things that could be improved. Why are there not lure modules for every type, or atleast most? It feels unfinished. It would be awesome for other players or friends to show up on your map, maybe not all the time, but if they were currently playing and toggled an option for it on. The map feels lonely, and less fleshed out.", "overall it's a really fun and entertaining game to play when you're bored. I also saw that the game is really improving in terms of lag in Go Battle League, as I have been seeing no lags so far. But maybe the game could have a system that saves the account details, so that you wouldn't have a chance of losing an account. I've lost like 2 accounts, and only retrieved one back so far :(", "I love this game. I'm just highly disappointed in the fact that only a few devices supports the AR mode for scanning Gyms and Pokéstops. Like is there no other app they can make that is compatible with most devices for AR scanning? Because my Samsung A02S is not even compatible for the AR scanning app and like I said, there are a few out of so many devices that are compatible.", "This game is amazing! You always have something to do or edit and the game gets updated frequently, 2 things, 1. is that it freezes sometimes. luckily, I don't care about that but it would be even better if you got rid of it and possibly other bugs, 2.when I update it it takes 600 Mb to remove apps or clear space which I find hard, I don't know if you can change that or not or its just the device if you can fix it, please make the MB lower. Thank you for reading :)", "I love Pokemon Go and have been immensely playing since it was launched. It's sad to find out that due to the latest update, I lost all the Special Research I've finished in the list. It served as our little trophy case especially for the ones who have been playing ever since. Would've given this 5☆s if not for that. Hoping it can still be retrieved and I will change my rating.", "Cool game overall, I would rate it 4 stars but the fact that the game's battling was unplayable on Samsung galaxy s7 for 2 full weeks is unacceptable. The update has finally rolled out to my phone, and so now I can get back to playing, but this experience has left a sour taste in my mouth. Start addressing bugs and your progress in the News feature, and I'll give this rating a second thought.", "Game's a blast! If you're needing a little extra motivation to get off the couch and go walking, this is the one! I would have given it 5 �, but their support sucks. Major issues and I reported  it in-game. Sometimes support will refund your lost items. Sometimes they just ignore you. Most recently, I had to face Giovanni. I get he's the main boss and should be hard, but I would have him beat and the screen would lock up. Only way to get it working again was to leave the match. St", '3 or 4 updates ago I lost my ability to copy/paste/edit text without deleting absolutely everything and starting over. I kept hoping it would come back but My screen is still full of selection menus.... And it makes deleting/sorting really tedius. Put options in the menu instead of clouding the entire screen with buttons. Otherwise, I love it!', 'It was a great game before, but it proceedingly has gotten worse over time, as it seems that the developers are trying to tear down the game themselves and sabotage it, littering in more and more expensive micro-transactions while giving little benifits for players. The game can still be fun, but raids seem to no longer be worth it anymore.', 'Thank you for helping us � I am disabled and it used to be easier to
enteract with gyms and poke stops, now you have to tresspass if you can even get close enough to play. As if that was not enough it gets upsetting to have to update every few days. No reason trying to play a game you can not even get to the main parts, not worth falling that often and just plain dangerous!', "I've played the game since it came out in 2016, but the fast rolling updates have made it so glitchy I can't participate in many of the game functions. Any type of battle (team go rocket, raids, or even battle league) has the charged attack or block feature glitching and not recognizing the taps. Additionally it has started to freeze and not transfer screens. After beating the latest legendary I had to restart the game because it froze and wouldn't let me capture. The glitches need to get fixed", 'I love this game, it is AMAZING!!! But there are a few things that I would like fixed: the map is a little out of date and I would like it if that would be updated. Sometimes I like to play pokemon go in the passenger seat ot the car, and when I drive past a pokestop and try to spin it, it always says try again later. I love that there are missions that you can do. It makes the game so much more fun and you can get cool prizes. Overall it is an amazing game and i definitely recommend it!!!', "This game has so much potential, but you can really tell Niantic just doesn't care. The amount of loading screens I run into, then I have to restart the game because my screen just goes black, or white. And stupid bugs that just make this game brutal, like completing objectives and it says you still haven't done them. And this hasn't just been going on for a couple weeks. I ran into these same issues when I played the game first in 2016. Very horrible production. Great game.", 'In the beginning, the game was fun and exciting to play but the years go by, it gets a bit boring and tedious. These days, the game just focusses on repeats (or recycle pokemons) which are not interesting anymore. It may be good for beginner players but the current senior players get bored easily.'], 'appId': 'com.nianticlabs.pokemongo', 'url': 'https://play.google.com/store/apps/details?id=com.nianticlabs.pokemongo&hl=en&gl=us'}
```

in line version ,it show 'version': 'Varies with device', not the right  value "0.245.2"

**Comment by FozerZ** [comment]
SOS! I too

**Comment by FozerZ** [comment]
i fix that #145

**Comment by JoMingyu** [comment]
It fixed from version [1.2.2](https://pypi.org/project/google-play-scraper/1.2.2/). special thanks to @FozerZ .
